server/server.py

```diff
@@ -18,7 +20,12 @@ def index():
 @app.route('/song', methods=['GET'])
```

Comment:
Can you make another endPoint '@song_wo_ytdata' that is the copy of the original '/song' endpoint, because the server will be unusable if the dev doesn't have an API key